pkg/kubelet/cm/topologymanager/socketmask/socketmask.go

```diff
@@ -0,0 +1,154 @@
+package socketmask
+
+import (
+	"strconv"
+        "strings"
+        "bytes"
+        "math"
+        "github.com/golang/glog"
+)
+
+
+type SocketMask []int64
```

Comment:
does the type has to be public?
if yes, this would need to have an export comment on top.